src/components/HotspotListItem.tsx

```diff
@@ -32,13 +33,32 @@ const HotspotListItem = ({
     const nextReward = await hntBalanceToDisplayVal(totalReward, false)
     setReward(`+${nextReward}`)
   }, [hntBalanceToDisplayVal, totalReward])
+  const { blockHeight } = useSelector((state: RootState) => state.heliumData)
 
   useEffect(() => {
     updateReward()
   }, [updateReward])
 
+  const percentSynced = useMemo(() => {
```

Comment:
I think this is being done in `HotspotDiagnosticReport` as well. Maybe we could create a hook for it?

Reply:
Created a util for it